app/src/androidTest/java/ch/epfl/sdp/appart/DatabaseTest.java

```diff
@@ -182,7 +182,7 @@ public void addingAdAndGetTest() throws IOException {
         Ad retrievedAd = db.getAd(card.getId()).join();
         verifyAd(retrievedAd, title, street, city, desc, price, globalUser.getUserId(), contactInfo, pricePeriod, hasVRTour);
 
-
+        database.removeFromStorage(database.getStorageReference(AdLayout.DIRECTORY + "/" + adId + "photo0.jpeg"));

```

Comment:
This looks a but more cumbersome but probably scales better.

```suggestion
                    database.removeFromStorage(database.getStorageReference(AdLayout.DIRECTORY + getApplicationContext().getResources().getString(R.string.SEPARATOR) + adId + "photo0.jpeg"));
```